starwars/src/components/Display.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './StarWars.css'
+
+function Display(props) {
+  console.log('props', props);
+  return (
+  <div className= "list-elements">
+    <ul>
+      <li>Name: {props.height.name}</li>
```

Comment:
This group of <li> elements is exactly on the dot. The only change that could be made is naming conventions since `props.height.name` is not describing the prop's data well. Right? Height doesn't usually have a name characteristic. `props.character.name` would be better.